docs/README.md

```diff
@@ -0,0 +1,90 @@
+# How to use Clever Tools
+
+Clever Tools is the command line interface (CLI) of Clever Cloud. You can use it to create and manage applications, addons and multiple services of the platform. It also provides an easy authenticated access to Clever Cloud public APIv2 and APIv4 through the [`clever curl` command](#curl).
```

Comment:
```suggestion
Clever Tools is the command line interface (CLI) of Clever Cloud. You can use it to create and manage multiple services of the platform as applications, databases or storage add-ons. It also provides an easy authenticated access to Clever Cloud public APIv2 and APIv4 through the [`clever curl` command](#curl).
```

Reply:
As previously discussed, this manual documentation is just a bootstrap, it will be automated in a next step. But we needed an up to date doc in the meantime.